### ISSUE TITLE
Tour of dgraph -> panic: interface conversion: interface {} is *time.Time, not time.Time

### ISSUE THREAD

**Issue by chrishiestand** [dgraph-io/dgraph#4824]
### What version of Dgraph are you using?
Dgraph version   : v1.2.1
Dgraph SHA-256   : 3f18ff84570b2944f4d75f6f508d55d902715c7ca2310799cc2991064eb046f8

### Have you tried reproducing the issue with the latest release?
Yes via docker image `dgraph/dgraph:latest`

The issue does not reproduce with dgraph v1.1.1

### What is the hardware spec (RAM, OS)?
32GiB, MacOS running Docker Desktop 2.2.0.3 (42716)

### Steps to reproduce the issue (command/config used to run Dgraph).
Following the steps here https://tour.dgraph.io/master/moredata/1/
1. Run the /alter endpoint reqeust
2. download `1million.rdf.gz` and the dgraph live loader:
```
docker exec -it dgraph dgraph live -f 1million.rdf.gz --alpha localhost:9080 --zero localhost:5080 -c 1
```

### Expected behaviour and actual result.
Expected: live load finishes successfully

Actual:
```
[Decoder]: Using assembly version of decoder
I0220 20:01:36.726591     162 init.go:98]

Dgraph version   : v1.2.1
Dgraph SHA-256   : 3f18ff84570b2944f4d75f6f508d55d902715c7ca2310799cc2991064eb046f8
Commit SHA-1     : ddcda9296
Commit timestamp : 2020-02-06 15:31:05 -0800
Branch           : HEAD
Go version       : go1.13.5

For Dgraph official documentation, visit https://docs.dgraph.io.
For discussions about Dgraph     , visit https://discuss.dgraph.io.
To say hi to the community       , visit https://dgraph.slack.com.

Licensed variously under the Apache Public License 2.0 and Dgraph Community License.
Copyright 2015-2018 Dgraph Labs, Inc.



Running transaction with dgraph endpoint: localhost:9080
Found 1 data file(s) to process
Processing data file "1million.rdf.gz"
panic: interface conversion: interface {} is *time.Time, not time.Time

goroutine 218 [running]:
github.com/dgraph-io/dgraph/tok.YearTokenizer.Tokens(0x18732a0, 0xc000f2ad40, 0x10, 0xc0045f9560, 0x14, 0xc0032627b3, 0xc000266a80)
	/tmp/go/src/github.com/dgraph-io/dgraph/tok/tok.go:214 +0x185
github.com/dgraph-io/dgraph/tok.BuildTokens(0x18732a0, 0xc000f2ad40, 0x1a864e0, 0x249ab50, 0x1a864e0, 0x249ab50, 0x0, 0x1, 0xc0045f9550)
	/tmp/go/src/github.com/dgraph-io/dgraph/tok/tok.go:104 +0x63
github.com/dgraph-io/dgraph/dgraph/cmd/live.(*loader).conflictKeysForNQuad(0xc00015c0e0, 0xc00324b5f0, 0x10, 0x2, 0x2, 0x0, 0x0)
	/tmp/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/live/batch.go:309 +0x49e
github.com/dgraph-io/dgraph/dgraph/cmd/live.(*loader).conflictKeysForReq(0xc00015c0e0, 0xc0028fdf88, 0xc000f92001, 0x6a7, 0x800)
	/tmp/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/live/batch.go:330 +0xe9
github.com/dgraph-io/dgraph/dgraph/cmd/live.(*loader).makeRequests(0xc00015c0e0)
	/tmp/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/live/batch.go:392 +0x1db
created by github.com/dgraph-io/dgraph/dgraph/cmd/live.setup
	/tmp/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/live/run.go:359 +0x2f0
```

**Comment by MichelDiz** [comment]
Hey, @chrishiestand thanks for the issue.

This is a known issue and we already have it fixed. Wait for a new release tho.

PR: https://github.com/dgraph-io/dgraph/pull/4793
Original issue: https://github.com/dgraph-io/dgraph/issues/4468

Cheers.

**Comment by MichelDiz** [comment]
Closing this

If you need this fix you can use the docker tag `v2.0.0-beta`
https://hub.docker.com/layers/dgraph/dgraph/v2.0.0-beta/images/sha256-f7ae12edd28b6603186e6af4924dfe1dd8ad008a27bf65f6e4d1b5477a61c1eb?context=explore

Also, you can always build binaries from the master by cloning the repo `cd /dgraph/dgraph` and then `make dgraph`.

**Comment by chrishiestand** [comment]
Thank you @MichelDiz, and sorry I missed the original issue.
